velox/vector/BaseVector.h

```diff
@@ -198,12 +198,19 @@ class BaseVector {
   /**
    * Returns a smart pointer to the null bitmap data for this
    * vector. May hold nullptr if there are no nulls. Not const because
-   * some vectors may generate this on first access.
+   * some vectors may generate this on first access. For ConstantVector, this
+   * method returns a BufferPtr of only size 1. For DictionaryVector, this
+   * method returns a BufferPtr for only nulls in the dictioanry layer.
    */
   const BufferPtr& nulls() const {
     return nulls_;
   }
 
+  // Returns a pointer to the raw null bitmap buffer of this vector. Notice that
+  // nulls of a ConstantVector or DictionaryVector should not be accessed
+  // directly through this API. If the vector is a ConstantVector, rawNulls_ is
+  // only of size 1. If the vector is a DictionaryVector, rawNulls_ doesn't
```

Comment:
If this is a Dictionary, it will return a point to the top-level nulls, no? Can we make the phrasing a bit more clear?